src/simulation/m_start_up.fpp

```diff
@@ -1209,15 +1219,16 @@ contains
         integer, intent(inout) :: nt
 
         integer :: i
+        real(kind(0d0)) :: dtdid, time_prev
 
         if (cfl_dt) then
-            if (cfl_const_dt .and. t_step == 0) call s_compute_dt()
+            if (cfl_const_dt .and. t_step == 0 .and. .not. lag_bubbles) call s_compute_dt()
 
-            if (cfl_adap_dt) call s_compute_dt()
+            if (cfl_adap_dt .and. .not. lag_bubbles) call s_compute_dt()
 
             if (t_step == 0) dt_init = dt
 
-            if (dt < 1d-3*dt_init .and. cfl_adap_dt .and. proc_rank == 0) then
+            if (dt < 1d-3*dt_init .and. cfl_adap_dt .and. proc_rank == 0 .and. .not. lag_bubbles) then
```

Comment:
can you do a find/replace on the whole code base to replace `lag_bubbles `with `EL_bubbles`

Reply:
I renamed the EL switch to bubbles_lagrange and also updated the original flag of the EE model from bubbles to bubbles_euler. Please confirm me if you want me to do some additional changes here.